src/Umbraco.Core/Models/ContentExtensions.cs

```diff
@@ -444,6 +434,19 @@ public static void PropertyValues(this IContent content, object value)
             }
         }
 
+        public static IContentTypeComposition GetContentType(this IContentBase contentBase)

```

Comment:
Ahh... I see this usage now, it's not an extension method and it doesn't require the singleton ApplicationContext.Current, but one could argue that this should be a Property instead of a method ?